lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -98,13 +99,10 @@ class FullPageScreenshot extends FRGatherer {
     await networkMonitor.enable();
 
     await session.sendCommand('Emulation.setDeviceMetricsOverride', {
-      // If we're gathering with mobile screenEmulation on (overlay scrollbars, etc), continue to use that for this screenshot.
-      mobile: context.settings.screenEmulation.mobile,
+      mobile: deviceMetrics.mobile,
       height,
-      width,
-      deviceScaleFactor: 1,
-      scale: 1,
-      screenOrientation: {angle: 0, type: 'portraitPrimary'},
+      width: 0, // Don't change
```

Comment:
using the 0 to not set width and dpr is INTERESTING. the protocol [sez](https://chromedevtools.github.io/devtools-protocol/tot/Emulation/#method-setDeviceMetricsOverride) 0 disables the override, and there is already the override we applied in emulation.js ... so i'm kinda unclear on if the 0 is clearing out any width/dpr override or its just not changing them on this call.  Do you have a sense?  
i found [this](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/public/common/widget/device_emulation_params.h;l=36;drc=36bef2fd0b71a5f17dd5dcee9241ad705c3569e2) which seems to indicate the don't-set behavior, at least for dpr. i don't see a similarish path for width.

Reply:
I definitely think it's the "don't change on this call" behavior. If it was the "clear current emulation" behavior, the viewport would revert back to some default at this point and I don't see that happening. The viewport width remains the same in all my testing.